app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,8 @@
 <resources>
     <string name="app_name">EpamAndroid</string>
+    <string name="button1">Button standard value</string>
```

Comment:
strange names. Better make name connected to value.

For example `button_standard_value`

Reply:
Please fix all string res names

Reply:
@egorikftp deleted it